04/app.js

```diff
@@ -0,0 +1,27 @@
+const divElements = document.querySelectorAll('div');
+
+if (!divElements || !divElements.length) {
+    throw new Error('No divs found!');
+}
+
+divElements.forEach(divElement =>
+    divElement.addEventListener('click', addClass)
+);
```

Comment:
Funkcje strzałkowe dopiero przy ES2015 - oczywiście jeśli mentee zna to nie zwracamy na to uwagi, ale sami nie wykorzystujemy ich :)

Reply:
Jasne :) nie używałem tłumacząc zadania